src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect } from 'react';
+import classnames from 'classnames';
+
+type Props = {
+  error: boolean,
+  setError: React.Dispatch<React.SetStateAction<boolean>>,
+};
+
+export const ErrorNotification: React.FC<Props> = ({ error, setError }) => {
+  useEffect(() => {
+    setTimeout(() => {
+      setError(false);
+    }, 3000);
```

Comment:
You should clear the timeout on the component unmount. You can read here about it https://bobbyhadz.com/blog/react-cleartimeout